app/browser/api/ledger.js

```diff
@@ -1340,7 +1340,9 @@ const enable = (state, paymentsEnabled) => {
 
     togglePromotionTimeoutId = setTimeout(() => {
       checkPromotions()
-    }, random.randomInt({min: 10 * ledgerUtil.milliseconds.second, max: 15 * ledgerUtil.milliseconds.second}))
+    }, process.env.LEDGER_ENVIRONMENT === 'staging'
+      ? random.randomInt({min: 10 * ledgerUtil.milliseconds.second, max: 15 * ledgerUtil.milliseconds.second})
+      : ledgerUtil.milliseconds.minute)
```

Comment:
I think we should capture this in a common variable (whether in `appConfig` or something scoped only to `ledger.js`) then we can share it between all of the needed delays

Reply:
This variable needs to be random so like 45 to 60s min

Reply:
@ryanml It was told to me in the past that the settings object was for settings exposed in preferences (I know there's things in there that don't adhere to that but I think we want to avoid adding more non-user-exposed items to settings). 

@NejcZdovc done.

Reply:
that's true, appConfig shouldn't have this timers in ther